test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature

```diff
@@ -1805,23 +1829,23 @@ Feature: update or append an attribute by entity ID in NGSI v2. "POST" - /v2/ent
       | metadatas_type   | alarm       |
       | metadatas_value  | hot         |
     And verify that receive several "Created" http code
-    When update or append an attribute by ID "room"
+    When update or append attributes by ID "room"
       | parameter       | value                  |
-      | attributes_name | temperature            |
+      | attributes_name | humidity               |
       | metadatas_name  | <attributes_meta_name> |
       | metadatas_value | 5678                   |
     Then verify that receive an "Created" http code
-    And verify that entities are stored in mongo
+    And verify that an entity is updated in mongo
     Examples:
       | attributes_meta_name |
       | 34                   |
-    #  | 34.4E-34             |
```

Comment:
As the test is now skipeed, maybe the line could be uncomented (same in two more cases, below).